packages/lib/ren-tx/src/machines/burn.ts

```diff
@@ -85,7 +85,9 @@ export interface BurnMachineSchema {
         created: {};
         createError: {};
         srcSettling: {};
+        errorBurning: {};
```

Comment:
I would be cool if you provide comments with a brief description of that states, as in DepositMachineSchema

Reply:
Can do